channels/drive/client/drive_file.c

```diff
@@ -563,6 +568,8 @@ BOOL drive_file_set_information(DRIVE_FILE* file, UINT32 FsInformationClass, UIN
 				if (m != st.st_mode)
 					fchmod(file->fd, st.st_mode);
 			}
+#else /* WIN32 */
+			SetFileTime(file->fd, &CreationTime, &LastAccessTime, &LastWriteTime);
```

Comment:
Not your code, but it would be nice to have ```SetFileTime``` implemented in winpr and the platform switch here eliminated.